ideascube/mediacenter/management/commands/clean.py

```diff
@@ -34,7 +34,13 @@ def add_arguments(self, parser):
         clean_media.set_defaults(func=self.clean_media)
 
     def clean_media(self, options):
-        Document.objects.filter(package_id='').delete()
+        documents = Document.objects.filter(package_id='')
+        documents_count = documents.count()
+
+        self.stdout.write(
```

Comment:
Maybe move the write after the `delete` ?
This way, if the `delete` fails for any reason and raise a exception, we don't print that x documents have been deleted.